Lukabaramishvili/imm-search-tool
# feat: create table w/ styles & fetch properties

## Description

This PR creates tables to display properties details. It fetches data from the api.js file provided by IMMO.
The table is dynamic and fully responsive.

## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

- Clone the repo
- install dependencies by running `npm install` or `yarn add`
- Start the project locally by doing `npm start` or `yarn start`
- Search the property by typing the full or partial name of the address and observe changes
